src/datapoints-graph/model/datapoints-graph-widget.model.ts

```diff
@@ -115,6 +116,31 @@ export interface DatapointWithValues extends DatapointsGraphKPIDetails {
   values: DatapointApiValues;
 }
 
+type DataPointModifiedValues = {
+  min: number;
+  max: number;
+};
+export type DpValuesArrayItem = {
```

Comment:
'Array' adds specific context to type that should be rather generic- if somebody wants to use this type as single value, it should be possible and indicating it's a element of array might be misleading
```suggestion
export type DpValuesItem = {
```